nodes.go

```diff
@@ -167,3 +173,59 @@ func (n *Node) Storage(name string) (storage *Storage, err error) {
 
 	return
 }
+
+//networks
+func (n *Node) Networks() (networks NodeNetworks, err error) {
+	err = n.client.Get(fmt.Sprintf("/nodes/%s/network", n.Name), &networks)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, v := range networks {
+		v.client = n.client
+		v.Node = n.Name
+		v.NodeApi = n
+	}
+
+	return
+}
+func (n *Node) Network(iface string) (network *NodeNetwork, err error) {
+
+	err = n.client.Get(fmt.Sprintf("/nodes/%s/network/%s", n.Name, iface), &network)
+	if err != nil {
+		return nil, err
+	}
+
+	if nil != network {
+		network.client = n.client
+		network.Node = n.Name
+		network.NodeApi = n
+		network.Iface = iface
+	}
+
+	return network, nil
+}
+
+func (n *Node) NewNetwork(network *NodeNetwork) error {
+
+	err := n.client.Post(fmt.Sprintf("/nodes/%s/network", n.Name), network, network)
+	if nil != err {
+		network = &NodeNetwork{}
```

Comment:
why did you add this here? i can't see it doing anything

Reply:
sorry， deleted this line now